chore(linter): simplify fixture test case to avoid conflicts when new rules are added

connorshea opened the pull request:
We could alternatiely fix this by censoring the # of rules emitted, but I don't know if that's really worthwhile. I believe this should be sufficient in that the rule will use its default values and thus still reproduces the issue correctly.

See #11644 for reference

camc314 reviewed (approved):
Thanks - could you see me struggling with CI haha

Sysix commented:
Love it! Thanks for fixing all the small problems which we did not cover because of missing resources  <3

connorshea committed:
chore(linter): simplify fixture test case to avoid conflicts when new rules are added (#17546)

We could alternatiely fix this by censoring the # of rules emitted, but I don't know if that's really worthwhile. I believe this should be sufficient in that the rule will use its default values and thus still reproduces the issue correctly.

See #11644 for reference
